src/tox_lsr/config_files/tox-default.ini

```diff
@@ -461,13 +461,33 @@ allowlist_externals = podman
     rm
 setenv =
     {[testenv]setenv}
-    GITHUB_OUTPUT = /tmp/mdl.log
-    INPUT_PATH = .
+    INPUT_CONFIG=.markdownlint.yaml
+    INPUT_FIX=true
 commands = bash {lsr_scriptdir}/setup_module_utils.sh
     {[lsr_config]commands_pre}
     {env:LSR_CONTAINER_RUNTIME:podman} run --rm --privileged \
       -v {toxinidir}:/workdir --workdir /workdir \
-      -e GITHUB_OUTPUT -e INPUT_PATH \
-      ghcr.io/actionshub/markdownlint:v3.1.3 \
+      -e INPUT_CONFIG -e INPUT_FIX \
+      docker://avtodev/markdown-lint:master \
```

Comment:
Using the same tool as in https://github.com/linux-system-roles/.github/pull/41/files#diff-0a5f55409408031dd08f66bce4e79a5ce01e2a241b989dfebef76274c348b836R31